src/main/webapp/index.html

```diff
@@ -20,7 +20,14 @@
         <label for="actions-radio" id="actions-label" class="menu-label">Action</label>
       </div>
       <div id="selected-option">
-        <div id="slogan"><p>Slogan</p></div>
+        <!--<div id="slogan"><p>Slogan</p></div>-->
+        <form action="/keyword" method="POST">
+          <textarea 
+            name="keyword-sentence" 
+            placeholder="Enter your tweet or interests here">Black Lives Matter and COVID-19.</textarea>
+          <br/>
+          <input type="submit" />
+        </form>
```

Comment:
I think this from should probably go in a different page. The user should log in or input the tweet in one page, and then get redirected to index.html where the keywords, news and actions are shown.

Reply:
I will move it to the keywords page for now.